SDVModTest/ModConfig.cs

```diff
@@ -85,5 +86,25 @@ class ModConfig
             new int[] { 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0 },
             new int[] { 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0 }
         };
+
+        public int[][] Scarecrow { get; set; } = CalculateScarecrow(17);
+
+        public int[][] DeluxeScarecrow { get; set; } = CalculateScarecrow(33);
+
+        private static int[][] CalculateScarecrow(int size)

```

Comment:
I love solving this with math! This function should be in the Tools class, please.

Reply:
I didn't realize there was a tools class. I will definitely add it there. Thanks!